src/services/plugin.service.ts

```diff
@@ -81,4 +82,35 @@ export class PluginService {
       return page;
     });
   }
+
+  public setPluginActive(plugin: Plugin | string, active: boolean) {
```

Comment:
Since you are only using the plugin name, it would be easier to just make the parameter a string. Especially since the only way this is called is through user input, which is already a string

Reply:
not really a fan of this name. can we do something like `setPluginState` or `setPluginStatus` ?